packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -426,7 +426,7 @@ export const ImageInput = ({
             variant="primary"
             aria-hidden="true"
             tabIndex={-1}
-            label=""
+            label="-" // We need to pass a label here to prevent IconButton from throwing
```

Comment:
If we pass the `label` here it causes problems with tests because `getByText(label)` will match multiple elements.

If this was only for Circuit specs it would be fine, but teams could also want to use `getByText` for their own specs and it would be unexpected that two elements match.